aspnetcore/signalr/hubs.md

```diff
@@ -120,6 +120,56 @@ By default, a server hub method name is the name of the .NET method. To change t
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/signalr/hubs/samples/6.x/SignalRHubsSample/Snippets/Hubs/ChatHub.cs" id="snippet_HubMethodName" highlight="1":::
 
+## Inject services into your hub
+
+> [!NOTE]
+> The following feature is only available in .NET 7.0+
```

Comment:
I recommend going ahead and versioning the whole topic at this point. Rick will activate it in the version selector. However, be sure to put the new 7.0 piece at the **_end of the topic_** to avoid broken links during preview.